modules/op-release-notes-1-18.adoc

```diff
@@ -0,0 +1,375 @@
++
+.Example annotation with commas
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: "comma-annotation"
+  annotations:
+    pipelinesascode.tekton.dev/on-target-branch: "[main, branchWith&#44;comma]"
+spec:
+# ...
+----
+
+* With this release, after a pull request is closed or merged, all associated `PipelineRun` resources that have the `cancel-in-progress: true` annotation are automatically canceled. To enable this feature, specify the `cancel-in-progress` annotation in your `PipelineRun` definition:
++
+.Example `cancel-in-progress` annotation
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: "cancel-in-progress"
+  annotations:
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+spec:
+# ...
+----
+
+* With this release, an active `PipelineRun` resource that has the `cancel-in-progress: true` annotation is automatically canceled when a new `PipelineRun` resource with the same name is triggered by {pac}. Now, after a pull request is raised and triggers a `PipelineRun`, and subsequent commits to the same pull request trigger a new `PipelineRun`, the older obsolete `PipelineRun` is canceled to save resources.
++
+To enable the feature, set the `pipelinesascode.tekton.dev/cancel-in-progress` annotation to `true` in the `PipelineRun` resource:
++
+.Example of enabling the cancellation of older pipeline runs
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: "cancel-in-progress"
+  annotations:
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+spec:
+# ...
+----
+
+* Before this update, the `.pathChanged()` function used in CEL expressions with Bitbucket Data Center did not work. With this release, the functionality is implemented.
+
+* With this release, you can match pipeline runs to pull requests by using labels. To match a pipeline run to a label, set the `pipelinesascode.tekton.dev/on-label` annotation in `PipelineRun` resources. Adding the label to a pull request or push event immediately triggers the pipeline run. If a push request or pull event still has the label and is updated with a commit, the pipeline run is triggered again.
++
+This functionality is supported on GitHub, GitLab, and Gitea repository providers. It is not supported on Bitbucket Cloud and Bitbucket Data Center.
+
+[id="tekton-results-new-features-1-18_{context}"]
+=== {tekton-results}
+
+* With this release, the Operator installs and adds configuration for {tekton-results} by default through the `TektonConfig` CR.
+
+* With this release, the {tekton-results} API server uses proxy environment variables in the {pipelines-shortname} console plugin to send authorization headers to the {tekton-results} API.
+
+* With this release, the {tekton-results} logging information fetched from LokiStack includes the container name for each log entry.
```

Comment:
For this sentence can we add a info like why container name was required for each entry 
I mean my point is adding reason for this change  would really helpful
cc @khrm

Reply:
@savitaashture can you provide the reason for this change? Time for release is very tight now.

Reply:
Container are either steps or some side containers of TaskRun.